cmd/mesh/main.go

```diff
@@ -11,16 +11,20 @@ import (
 	"sync"
 
 	"github.com/0xProject/0x-mesh/core"
+	"github.com/0xProject/0x-mesh/rpc"
 	"github.com/plaid/go-envvar/envvar"
 	log "github.com/sirupsen/logrus"
 )
 
 // standaloneConfig contains configuration options specific to running 0x Mesh
 // in standalone mode (i.e. not in a browser).
 type standaloneConfig struct {
-	// RPCAddr is the interface and port to use for the JSON-RPC API over
+	// WSRPCAddr is the interface and port to use for the JSON-RPC API over
 	// WebSockets. By default, 0x Mesh will listen on localhost and port 60557.
-	RPCAddr string `envvar:"RPC_ADDR" default:"localhost:60557"`
+	WSRPCAddr string `envvar:"WS_RPC_ADDR" default:"localhost:60557"`
+	// HTTPRPCAddr is the interface and port to use for the JSON-RPC API over
+	// HTTP. By default, 0x Mesh will listen on localhost and port 60556.
+	HTTPRPCAddr string `envvar:"HTTP_RPC_ADDR" default:"localhost:60556"`
```

Comment:
Is there a different way that we can write this variable? I can parse `WSRPCAddr`, but the `P` and the `R` in `HTTPRPCAddr` make it harder to read quickly. Maybe `HttpRPCAddr`?

Reply:
@jalextowle as annoying as this naming convention can be, it is a widely accepted Go standard and we should not change it. See https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Fair enough. I wish that wasn't the style, but I do agree that we should follow the standard.